docs/nginx-ingress.md

```diff
@@ -0,0 +1,105 @@
+# Ingress Configuration [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io)
+
+Ingress exposes HTTP and HTTPS routes from outside the cluster to services within the cluster. Traffic routing is controlled by rules defined on the Ingress resource. This document explains how to enable Ingress for a MinIO Tenant using the [Nginx Ingress Controller](https://kubernetes.github.io/ingress-nginx/).
+
+## Getting Started
+
+### Prerequisites
+
+- MinIO Operator up and running as explained in the [document here](https://docs.min.io/minio/k8s/deployment/deploy-minio-operator.html).
+- Nginx Ingress Controller installed and running as explained [here](https://kubernetes.github.io/ingress-nginx/deploy/).
+
+- Network routing rules that enable external client access to Kubernetes worker nodes. For example, this tutorial assumes `minio.example.net` as an externally resolvable URL. 
```

Comment:
```suggestion
- Network routing rules that enable external client access to Kubernetes worker nodes. For example, this tutorial assumes `minio.example.net` as an externally resolvable URL. 

```

Adding line return, not sure if markdown will be upset or not but it looks better either way.

Reply:
The tutorial actually assumes `minio.example.com` not `example.net`